app/src/main/java/com/example/concordia_campus_guide/Activities/MainActivity.java

```diff
@@ -39,6 +49,28 @@ protected void onCreate(Bundle savedInstanceState) {
 
         View infoCard = findViewById(R.id.info_card);
         swipeableInfoCard = BottomSheetBehavior.from(infoCard);
+
+        getApplication().getApplicationContext().deleteDatabase(AppDatabase.DB_NAME);
+
+        //load buildings
+        Buildings buildings = ApplicationState.getInstance(this).getBuildings();
+        AppDatabase appDb = AppDatabase.getInstance(this);
+        appDb.buildingDao().insertAll(buildings.getBuildings());
+
+        //load floors
+        Floors floors = ApplicationState.getInstance(this).getFloors();
+        appDb.floorDao().insertAll(floors.getFloors());
+
+        //load rooms
+        Rooms rooms = ApplicationState.getInstance(this).getRooms();
+        appDb.roomDao().insertAll(rooms.getRooms());
+
+
+        AppDatabase appDB = AppDatabase.getInstance(this);
+        List<Building> listBuildings = appDB.buildingDao().getAll();
```

Comment:
for now those are just to show example of how to get the info from DB right?

Reply:
yes